src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,69 @@
+import { useContext } from 'react';
+import classNames from 'classnames';
+import { FilterOption, TodoContext } from '../../context/TodoContext';
+
+export const TodoFooter = () => {
+  const {
+    filter, setFilter, visibleTodos, activeTodosAmount, clearCompleted,
+  } = useContext(TodoContext);
+
+  const completedTodosAmount = visibleTodos.some(todo => todo.completed);
+
+  const handleClearAllCompleted = () => {
+    if (completedTodosAmount) {
+      clearCompleted();
+    }
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodosAmount} items left`}
```

Comment:
If there is one item, there should not be a plural in the text.

Reply:
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/54949928/d157cc6c-f5cf-4150-a459-986b5ebc9ff2)
=/